pkg/monitor/intervalcreation/podlogs_test.go

```diff
@@ -0,0 +1,114 @@
+package intervalcreation
+
+import (
+	"testing"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+
+	kapi "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/openshift/origin/pkg/monitor/monitorapi"
+)
+
+// TODO: temporary test, remove before merge
```

Comment:
True comment?

Reply:
Oops, yes it's accurate, thank you, I'll get that test out.

Reply:
Updated

Reply:
:+1: